### ISSUE TITLE
Bug: create-rule.ts has incorrect GitHub URL

### ISSUE THREAD

**Issue by pertrai1** [pertrai1/eslint-plugin-llm-core#28]
## Description

In `src/utils/create-rule.ts`, the rule documentation URL points to:

```ts
`https://github.com/rsimpson2/eslint-plugin-llm-core/blob/main/docs/rules/${name}.md`
```

But the actual repository is `pertrai1/eslint-plugin-llm-core`.

## Impact

Links in IDEs and documentation will 404.

## Fix

Update to:
```ts
`https://github.com/pertrai1/eslint-plugin-llm-core/blob/main/docs/rules/${name}.md`
```
